debian/rules

```diff
@@ -51,7 +49,7 @@ override_dh_shlibdeps:
 override_dh_virtualenv:
 	./debian/build_virtualenv
 
-# We are restricted to compat level 9 (because xenial), so have to
-# enable the systemd bits manually.
 %:
-	dh $@ --with python-virtualenv --with systemd
+	# TODO: Experiment with --buildsystem=dh_virtualenv
+	# https://dh-virtualenv.readthedocs.io/en/latest/usage.html#experimental-buildsystem-support
```

Comment:
Note that including comments here, rather than above the `%:`, will cause them to be printed out when the deb is built, e.g.:

```
dpkg-buildpackage: info: host architecture amd64
 dpkg-source --before-build build
 debian/rules clean
# TODO: Experiment with --buildsystem=dh_virtualenv
# https://dh-virtualenv.readthedocs.io/en/latest/usage.html#experimental-buildsystem-support
dh clean --with python-virtualenv --buildsystem=pybuild
   dh_clean -O--buildsystem=pybuild
 dpkg-source -b build
```